code/modules/reagents/reagent_containers/food/condiment.dm

```diff
@@ -167,7 +167,7 @@
 				icon_state = SUGAR
 			if(CHEFSPECIAL)
 				name = "\improper Chef Excellence's Special Sauce"
-				desc = "A potent sauce distilled from the toxin glands of 1000 Space Carp."
+				desc = "A potent sauce distilled from the toxin glands of 1000 Space Carp with an extra touch of LSD, because why not?"
```

Comment:
See https://github.com/d3athrow/vgstation13/blob/Bleeding-Edge/code/modules/reagents/reagent_containers/food/condiment.dm#L254

That's the original description that no one ever saw because it gets replaced by the shorter one as soon as the reagent is added.